pkg/kubelet/prober/worker_test.go

```diff
@@ -530,6 +530,70 @@ func TestResultRunOnStartupCheckFailure(t *testing.T) {
 	}
 }
 
+func TestDoProbe_SidecarContainerWithRestartPolicy(t *testing.T) {
```

Comment:
can we use more specifc name for this as well?

`TestDoProbe_TerminatedRestartableInitContainerWithRestartPolicyNone` ?